ast/Trees.h

```diff
@@ -148,8 +148,11 @@ class MethodDef final : public Declaration {
     u4 flags;
 
     enum Flags {
```

Comment:
Is this worth turning into an `enum class`?

Reply:
Answering my own question: it does not, as it won't work with bitwise operators if it's an `enum class`.